portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+window.addEventListener('load', function () {
+    addRandomBackground();
+});
+
+
 /**
- * Adds a random greeting to the page.
+ * Adds a random backgournd wallpaper to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomBackground() {
+ const wallpapers = ['https://www.toptal.com/designers/subtlepatterns/patterns/y-so-serious-white.png',
+    'https://www.toptal.com/designers/subtlepatterns/patterns/webb.png'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random wallpaper.
+  const wallpaper = wallpapers[Math.floor(Math.random() * wallpapers.length)];
+console.log(document.body);
```

Comment:
You left your test code here, please remove them

Reply:
Fixed it.